File: java/src/main/java/pl/kkurczewski/json/value/ValueObjectController.java
package pl.kkurczewski.json.value;

import pl.kkurczewski.json.value.dto.Wrapper;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class ValueObjectController {

    @PostMapping("/wrapper")
    public Wrapper wrapper(@RequestBody Wrapper wrapper) {
        return wrapper;
    }
}

File: kotlin/src/main/kotlin/pl/kkurczewski/json/value/ValueObjectController.kt
package pl.kkurczewski.json.value

import pl.kkurczewski.json.value.dto.Wrapper
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RestController

@RestController
class ValueObjectController {
    @PostMapping("/wrapper")
    fun wrapper(@RequestBody wrapper: Wrapper) = wrapper
}